blockchain/blockchain-wallet-v4-frontend
# feat(BCH): allow importing bch address to kvstore

## Description (optional)
https://blockchain.atlassian.net/browse/FWLT-76

## Testing Steps (optional)
@seanblockchain you should be able to use bch kvStore action `importLegacyAddress`. The action takes a key (public or private key) and will either set the addr to bch kvStore or through an error if the addr type is not supported.
